packages/nekoton_repository/lib/src/models/seed.dart

```diff
@@ -27,6 +28,10 @@ class Seed extends Equatable {
   /// Just for iterating over all keys.
   List<SeedKey> get allKeys => [masterKey, ...subKeys];
 
+  /// Get instance of SeedKey if [publicKey] is part of this seed.
+  SeedKey? getKeyByPublicKey(String publicKey) =>
```

Comment:
Is it a really good idea to return null? Maybe we should throw error here instead?